src/ProcessManagerBundle/Installer.php

```diff
@@ -30,7 +30,6 @@ public function migrateInstall(Schema $schema, Version $version)
         $processTable = $schema->createTable('process_manager_processes');
         $processTable->addColumn('id', 'integer')
             ->setAutoincrement(true);
-        $processTable->addColumn('type', 'string');
```

Comment:
why? thats a new needed column

Reply:
Yes, but it is added in next migration. So:
1. If one install a new bundle, these two migrations do the job
2. If one has already installed previous version, second migration adds the column.

If the column addition is both in installer and migration there is the error: column type already exists and the bundle cannot be updated.

In migration there can be a check if the column exists but what about down() method? Shoul it remove the column or not.

So, it seems to me, that this is a proper way.

Reply:
ok, makes sense.